spiffworkflow-backend/src/spiffworkflow_backend/config/permissions/demo.yml

```diff
@@ -0,0 +1,88 @@
+default_group: everybody
```

Comment:
if we call this demo.yml, i think it's going to override for https://demo.spiffworkflow.org. we would ideally make a new environment like https://status.dev.spiffworkflow.org (and https://status.staging.spiffworkflow.org).

Reply:
renamed it to example.yml for now to avoid conflicts, and we can deal with new deployment configurations when the time comes.